README.md

```diff
@@ -0,0 +1,53 @@
+# restbase-mod-table-sqlite [![Build Status](https://travis-ci.org/wikimedia/restbase-mod-table-sqlite.svg?branch=master)](https://travis-ci.org/wikimedia/restbase-mod-table-sqlite)
+
+An SQLite3 back-end module for [RESTBase](wikimedia/restbase) conforming to the
+[RESTBase storage specification](wikimedia/restbase-mod-table-spec).
+
+## Installation
+
+Firstly, install RESTBase. The SQLite back-end module should be pulled in
+automatically as a dependency. If you cannot find `restbase-mod-table-sqlite` in
```

Comment:
Hm, I wouldn't actually do so, as that means introducing a binary dependency (and one on SQLite header files) when otherwise it is not needed, e.g. people that want the Cassandra back-end. Those that do not want the extra dev && test stuff, can use `npm install --production`. Conversely, `restbase-mod-table-cassandra` does not depend on header files and such, so it can be painfully installed even if it goes unused.